Modules/Module_Processing/Module_Population/xASL_stat_GetROIstatistics.m

```diff
@@ -240,18 +240,28 @@
 %if x.S.IsASL
     %% ------------------------------------------------------------------------------------------------------------
     %% 1. For all ROIs, skip ROIs smaller than 1 mL (296 voxels @ 1.5x1.5x1.5 mm)
-	if x.S.InputNativeSpace
+	if ~isfield(x.S, 'MinimalROIVolume') % We use this user-specific field by default
+        x.S.MinimalROIVolume = 1; % 1 mL
+    end
+    
+    if x.S.InputNativeSpace
 		VoxelSize = xASL_io_ReadNifti(fullfile(x.dir.xASLDerivatives,x.SUBJECTS{1},listSessions{1},[x.S.InputAtlasNativeName '.nii']));
 		VoxelSize = [norm(VoxelSize.mat(1:3,1)), norm(VoxelSize.mat(1:3,2)), norm(VoxelSize.mat(1:3,3))];
 	else
 		VoxelSize = [1.5 1.5 1.5];
 	end
 	if x.S.bSubjectSpecificROI
 		MinVoxels = 0; % For Lesions and ROIs we don't set a minimal ROI size
-	else
-		MinVoxels = 1000/prod(VoxelSize); % 1 mL
+    else
+
+		MinVoxels = round(1000*x.S.MinimalROIVolume/prod(VoxelSize)); % 1 mL (e.g., 296 voxels @ 1.5x1.5x1.5 mm)
 	end
 
+    fprintf('\n\n');
+    fprintf('%s\n', ['Assuming voxel size [' num2str(VoxelSize(1)) ' ' num2str(VoxelSize(2)) ' ' num2str(VoxelSize(3)) '] mm']);
+    fprintf('%s\n', ['Using a minimal ROI volume of ' num2str(x.S.MinimalROIVolume) ' mL, or a minimal number of ' xASL_num2str(MinVoxels) ' voxels']);

```

Comment:
please rephrase. it's not 'or', it's the same threshold in different units.

Reply:
That's English :) But you're right that it can be confusing. Fixed now in 46c421bcd1aed8156cbe146a8cf708930de3d337